CHANGELOG.md

```diff
@@ -2,6 +2,65 @@
 
 ## Unreleased
 
+### Migration notes
+
+- Rename HTTP configuration settings,
+  ([#8758](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8758))
+  - `otel.instrumentation.http.capture-headers.client.request`
+    → `otel.instrumentation.http.client.capture-request-headers`
+  - `otel.instrumentation.http.capture-headers.client.response`
+    → `otel.instrumentation.http.client.capture-response-headers`
+  - `otel.instrumentation.http.capture-headers.server.request`
+    → `otel.instrumentation.http.server.capture-request-headers`
+  - `otel.instrumentation.http.capture-headers.server.response`
+    → `otel.instrumentation.http.server.capture-response-headers`
+
+### 📈 Enhancements
+
+- Support latest armeria release
+  ([#8745](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8745))
+- Support latest mongo release
+  ([#8785](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8785))
+- Remove `server.{address,port}` from HTTP server metrics
+  ([#8771](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8771))
+- aws-sdk-2.2.: Support injection into SQS.SendMessageBatch message attributes
+  ([#8798](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8798))
+- Log4j and Logback appenders opt-in to using GlobalOpenTelemetry
+  ([#8791](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8791))
+- aws-sdk-2.2: SNS.Publish support with experimental messaging propagator flag
+  ([#8830](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8830))
+- support for adding baggage to log4j 2 ContextData
+  ([#8810](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8810))
+- Micrometer bridge: interpret no SLO config as no buckets advice
+  ([#8856](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8856))
+- Instrumentation for Elasticsearch 8+
+  ([#8799](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8799))
+- Add support for schemaUrls auto-computed from `AttributesExtrator`s
+  ([#8864](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8864))
+- Initialize appenders in the spring boot starter
+  ([#8888](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8888))
+- Support reactor-netty 1.0.34+
```

Comment:
while looking into this I noticed that our `reactor-netty-1.0` instrumentation does not work (or at least we don't test it with) reactor netty `1.1`